src/Tutor.Web/Mappings/Analytics/AiStatisticsDto.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Tutor.Web.Mappings.Analytics;
+
+public class AiStatisticsDto
+{
+    public int KcId { get; set; }
+    public int AiId { get; set; }
+    public int TotalCompleted { get; set; }
+    public int TotalPassed { get; set; }
+    public List<double> MinutesToCompletion { get; set; }
+    public List<int> AttemptsToPass { get; set; }
+
+    public override int GetHashCode()
```

Comment:
The reason I added this is to simplify the tests - instead of comparing the equality of individual DTO fields received from the system, I just type "actualObject.ShouldBe(expectedObject);"

Still need to develop tests for this particular feature though.